Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -669,6 +670,9 @@ ppExplanation (UnknownArch unknownArches) =
     "Unknown architecture name " ++ commaSep (map quote unknownArches)
 ppExplanation (UnknownCompiler unknownImpls) =
     "Unknown compiler name " ++ commaSep (map quote unknownImpls)
+ppExplanation (MissingUpperBounds pname) =
+  "'" ++ unPackageName pname ++ "' misses upper bounds, add them"
```

Comment:
Maybe a gentler "consider adding them with...", since for now it's just a `PackageDistSuspiciousWarn`. Also I wonder if we should add (a link to) a longer explanation of the problem